Makefile

```diff
@@ -20,11 +20,21 @@ ARCH ?= $(shell uname -m)
 DEBUG ?=
 LIBC ?= gnu
 KBC ?=
+ttrpc ?=
 DESTDIR ?= $(PREFIX)/bin
 RUSTFLAGS_ARGS ?=
+features ?=
 
 ifdef KBC
-    feature := --no-default-features --features
+    features += $(KBC)
+else
+    features += sample_kbc
+endif
+
+ifdef ttrpc
+    features += ttrpc
+else
+    features += grpc
```

Comment:
Is it supported to enable both `ttrpc` and `grpc`?

Reply:
Supported it now, thanks.